releases/data-center/syskit-point-2023-5/point-2023-5-release-note.md

```diff
@@ -24,8 +24,21 @@ See detailed step-by-step instructions on how to upgrade Syskit Point in the [up
 
 ## Features
 
-* **New options added to the Storage Management feature**!
-  * The Site Storage Metrics report now has the **Clean Up File Versions action**. 
+* **The storage usage reporting comes with a new Site Storage Metrics report**!
+  * You can drill from the Storage Metrics report to a more detailed per-site report by clicking the workspace name. 
+
+* **New File Storage Details** report is available!
+    * Drill from the Site Storage Metrics report to the File Storage Details report by clicking a file name.
+    * Use this report to see all file versions and the total storage used by all versions.
+    * Find how much storage is used by the file versions older than 30 days.
```

Comment:
provjeri ove brojke s Matejem - ovo mislim niti nisu defaulti što je trenutno podešeno na QA official. 
stavio bi tu prave defaulte i svakako linkao na configurtion article za storage management gdje se te brojke mogu mijenjati